// Copyright (c) 2010 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Internal message types that should not be seen outside the protocol
// directory.

syntax = "proto2";

import "auth.proto";
import "control.proto";
import "event.proto";

option optimize_for = LITE_RUNTIME;

package remoting.protocol;

// Represents a message being sent on the control channel.
message ControlMessage {
  optional SuggestResolutionRequest suggest_resolution = 1;
  optional NotifyResolutionRequest notify_resolution = 2;
  optional BeginSessionRequest begin_session_request = 3;
  optional BeginSessionResponse begin_session_response = 4;
}
